mopidy_alsamixer/mixer.py

```diff
@@ -70,15 +75,34 @@ def get_volume(self):
         if not channels:
             return None
         elif channels.count(channels[0]) == len(channels):
-            return int(channels[0])
+            return self.mixer_volume_to_volume(channels[0])
         else:
             # Not all channels have the same volume
             return None
 
     def set_volume(self, volume):
-        self._mixer.setvolume(volume)
+        self._mixer.setvolume(self.volume_to_mixer_volume(volume))
         return True
 
+    def mixer_volume_to_volume(self, mixer_volume):
+        volume = mixer_volume
+        if self.volume_scale == "cubic":
+            volume = GstAudio.StreamVolume.convert_volume(GstAudio.StreamVolumeFormat.CUBIC, GstAudio.StreamVolumeFormat.LINEAR, volume / 100.0) * 100.0
+        elif self.volume_scale == "db":
+            volume = math.pow(10, volume / 50.0)
+            #volume = GstAudio.StreamVolume.convert_volume(GstAudio.StreamVolumeFormat.DB, GstAudio.StreamVolumeFormat.LINEAR, volume / 100.0) * 100.0
```

Comment:
I would just remove "dead" code like this, and maybe add a quick comment about why we calculate this ourselves.